# ==============================================================================
# REGRESSOR COMMIT DIFF
# ==============================================================================
# File: gfx/layers/client/TextureClient.h
# Commit: bbfe23c61add
# Full Hash: bbfe23c61add07377daf80696fd6996c0463f5b6
# Author: Jeff Gilbert <jgilbert@mozilla.com>
# Date: 2020-06-09 09:21:34
# Regressor Bug: 1632249
# File Overlap Count: 3
# Description:
#   Bug 1632249 - Support compositing out-of-process WebGL. r=handyman,lsalzman,nical,geckoview-reviewers,agi,imanol
#   
#   * Majorly simplity CanvasRenderer
#   * Replace GLScreenBuffer with trivial GLSwapChain
#   * Use descriptor structs so that future SharedSurface changes aren't so painful
# ==============================================================================

diff -r 8fcfac450ecc -r bbfe23c61add gfx/layers/client/TextureClient.h
--- a/gfx/layers/client/TextureClient.h	Mon Jun 08 20:13:01 2020 +0000
+++ b/gfx/layers/client/TextureClient.h	Mon Jun 08 20:34:15 2020 +0000
@@ -335,7 +335,7 @@
  * lifetime. This means that the lifetime of the underlying shared data
  * matches the lifetime of the TextureClient/Host pair. It also means
  * TextureClient/Host do not implement double buffering, which is the
- * responsibility of the compositable (which would use two Texture pairs).
+ * responsibility of the compositable (which would use pairs of Textures).
  * In order to send several different buffers to the compositor side, use
  * several TextureClients.
  */
@@ -907,6 +907,8 @@
 bool UpdateYCbCrTextureClient(TextureClient* aTexture,
                               const PlanarYCbCrData& aData);
 
+TextureType PreferredCanvasTextureType(const KnowsCompositor&);
+
 }  // namespace layers
 }  // namespace mozilla
 